tests/test_client.py

```diff
@@ -322,7 +323,7 @@ def _create_batch_inventory_adjustment(self, start_date, num_records=1):
         for change_chunk in chunks(changes, 100):
             body = {
                 'changes': change_chunk,
-                'ignore_unchanged_counts': random.choice([True, False]),
+                'ignore_unchanged_counts': False,
```

Comment:
Do we still want this one? Not sure if it had any effect.

Reply:
I don't know, but literally telling telling the api to inconsistently report records seems like a bad idea to me. I want to leave it like this... unless @kspeer825 feels strongly otherwise.

Reply:
Agreed, False always seems better.